src/Symfony/Component/VarDumper/Dumper/CliDumper.php

```diff
@@ -636,6 +634,36 @@ private function hasColorSupport($stream): bool
         return preg_match('/^((screen|xterm|vt100|vt220|putty|rxvt|ansi|cygwin|linux).*)|(.*-256(color)?(-bce)?)$/', $term);
     }
 
+    /**
+     * Checks if the stream is a TTY, i.e; whether the output stream is connected to a terminal.
+     *
+     * Reference: Composer\Util\Platform::isTty
+     * https://github.com/composer/composer
+     */
+    private static function isTty(): bool
+    {
+        // Detect msysgit/mingw and assume this is a tty because detection
+        // does not work correctly, see https://github.com/composer/composer/issues/9690
+        if (\in_array(strtoupper((string) getenv('MSYSTEM')), ['MINGW32', 'MINGW64'], true)) {
+            return true;
+        }
+
+        // Modern cross-platform function, includes the fstat fallback so if it is present we trust it
+        if (\function_exists('stream_isatty')) {
```

Comment:
we can and should remove the isTty method because this very check is always true on PHP 7.1, so the resulting code can be inlined
In the phpunit-bridge, we cannot yet because it still needs to support 7.1

Reply:
Are you sure? https://www.php.net/manual/en/function.stream-isatty says it was introduced in PHP 7.2.